docs/website/docs/plus/getting-started/tutorial.md

```diff
@@ -0,0 +1,395 @@
+│   ├── config.toml
+│   ├── dev.secrets.toml
+│   └── secrets.toml
+├── _storage/             # local storage for your project, excluded from git
+├── sources/              # your sources, contains the code for the arrow source
+│   └── arrow.py
+├── .gitignore
+├── requirements.txt
+└── dlt.yml       # the main project manifest
+```
+
+### Understanding `dlt.yml`
+The `dlt.yml` file is the central configuration for your dlt+ Project. It defines the pipelines, sources, and destinations. In the generated project, the file looks like this:
+
+```yaml
+profiles:
+  # profiles allow you to configure different settings for different environments
+  dev: {}
+
+# your sources are the data sources you want to load from
+sources:
+  arrow:
+    type: sources.arrow.source
+
+# your destinations are the databases where your data will be saved
+destinations:
+  duckdb:
+    type: duckdb
+
+# your datasets are the datasets on your destinations where your data will go
+datasets: {}
+
+# your pipelines orchestrate data loading actions
+pipelines:
+  my_pipeline:
+    source: arrow
+    destination: duckdb
+    dataset_name: my_pipeline_dataset
+```
+
+:::tip
+If you do not want to start with a source, destination, and pipeline, you can simply run `dlt project init tutorial`, where `tutorial` is the name of your dlt+ Project. This will generate a project with empty sources, destinations, and pipelines.
+:::
+
+Some details about the project structure above:
+
+* The `project` section could be fully omitted in this case; it will be generated to be more explicit what the default settings are.
+* The `runtime` section is analogous to the config.toml [runtime] section and could also be omitted in this case.
+* The `profiles` section is not doing much in this case. There are two implicit profiles: `dev` and `tests` that are present in any project; we will learn about profiles in more detail later.
+
+You can reference environment variables in the `dlt.yml` file using the `${ENV_VARIABLE_NAME}` syntax. Additionally, dlt+ provides several [predefined project variables](../features/projects.md#substitution) that are automatically substituted during loading.
+
+:::tip
+You can find more information about the `dlt.yml` structure in the [dlt+ Project section](../core-concepts/project.md).
+:::
+
+## Running the pipeline
+
+Once the project is initialized, you can run the pipeline using:
+
+```sh
+dlt pipeline my_pipeline run
+```
+
+This command:
+- Locates the pipeline named `my_pipeline` in `dlt.yml`.
+- Executes it, populating the duckdb destination that is defined to be stored in `${tmp_dir}my_data.duckdb`.
+
+:::tip
+Take a look at the [Projects context](../features/projects.md#project-context) to learn more about how to work with nested projects and how dlt searches for the pipelines based on its name.
+:::
+
+### Inspecting the results
+
+Use the `dlt dataset` command to interact with the dataset stored in the DuckDB destination. For example:
```

Comment:
link to cli ref?

Reply:
fixed in https://github.com/dlt-hub/dlt/pull/2325